main.tex

```diff
@@ -239,9 +239,11 @@ \subsection{Astropy development model}
 line count includes comments, as these are often as important for
 maintainability as the code itself.  Without comments there are $142197$ lines
 of code.} contributed by $232$ unique contributors over $19270$ \texttt{git}
-commits.
-The commit distribution \inlinecomment{Tom R}{too vague, maybe 'the distribution of number of commits as a function of ...'} follows an approximate power-law (with log-slope of
-$\approx -0.47$) \inlinecomment{AG}{round to -0.5?  This is astronomy, after all.}, meaning most of the commits were and are contributed by a
+commits. We can construct a distribution of the number of commits
+per contributor ordered by decreasing number of commits. When doing this,
+we find that this distribution
+follows an approximate power-law (with log-slope of
+$\approx -0.5$), meaning most of the commits were and are contributed by a
```

Comment:
Just to illustrate why this number is confusing and (I think) unnecessary: I'm wondering whether this -0.5 is the slope of the decreasing bins in `log(dN / dlog(Ncommits))` vs. `log(Ncommits)` space (is that what you mean by log-slope?), or if it is the actual index of the power-law distribution? (which differs by +1)